styles/var.css

```diff
@@ -1,22 +1,24 @@
 /* Generic colors */
 :root {
   --lnk-hvr: #00aeff;
+  --green: #CCFF66;
+  --green-rgb: 204, 255, 102;
```

Comment:
could these be redundant?

Reply:
not quite, the rgb version is used for transparency
i.e. rgba(var(--green-rgb), [insert alpha value here])
if you used it as is it wouldn't work